docs/how-to/multicluster-gateways-walkthrough.md

```diff
@@ -13,13 +13,7 @@ We will start with a single cluster and move to multiple clusters to illustrate
 
 ## Initial Setup
 
-In this walkthrough, we'll deploy test echo services across multiple clusters. Before starting, set up a `MGC_SUB_DOMAIN` variable to simplify templating. This variable will represent the host for each service.
-
-If you followed the [Getting Started Guide](https://docs.kuadrant.io/getting-started/), you would have already set up a `MGC_ZONE_ROOT_DOMAIN` environment variable when creating a `ManagedZone`. For this tutorial, we'll derive a host from this domain.
-
-```bash
-export MGC_SUB_DOMAIN=myapp.$MGC_ZONE_ROOT_DOMAIN
-```
+In this walkthrough, we'll deploy test echo services across multiple clusters. If you followed the [Getting Started Guide](https://docs.kuadrant.io/getting-started/), you would have already set up a `MGC_ZONE_ROOT_DOMAIN` environment variable when creating a `ManagedZone`. For this tutorial, we'll derive a host from this domain for these echo services.
```

Comment:
```suggestion
In this walkthrough, we'll deploy test echo services across multiple clusters. If you followed the [Getting Started Guide](https://docs.kuadrant.io/getting-started/), you would have already set up a `MGC_ZONE_ROOT_DOMAIN` environment variable. For this tutorial, we'll derive a host from this domain for these echo services.
```
The user itself doesn't set up the Manage Zone. The script does it so i'm thinking the inclusion of that might cause some confusion as to an additional step that might need to be taken as part of getting started.